### ISSUE TITLE
CHANGELOG.md missing at install via pip

### ISSUE THREAD

**Issue by faemmi** [ecmwf/pyodc#5]
Trying to install `pyodc 1.1.0` via `pip` gives the following error:
```commandline
$ pip install pyodc==1.1.0
Collecting pyodc==1.1.0
  Using cached pyodc-1.1.0.tar.gz (25 kB)
  Installing build dependencies ... done
  Getting requirements to build wheel ... error
  ERROR: Command errored out with exit status 1:
   command: /home/fabian/anaconda3/envs/test/bin/python /home/fabian/anaconda3/envs/test/lib/python3.7/site-packages/pip/_vendor/pep517/in_process/_in_process.py get_requires_for_build_wheel /tmp/tmpiv0acpiq
       cwd: /tmp/pip-install-609l767c/pyodc_be4b799dbebc454f9d42773def8ae366
  Complete output (20 lines):
  Traceback (most recent call last):
    File "/home/fabian/anaconda3/envs/test/lib/python3.7/site-packages/pip/_vendor/pep517/in_process/_in_process.py", line 349, in <module>
      main()
    File "/home/fabian/anaconda3/envs/test/lib/python3.7/site-packages/pip/_vendor/pep517/in_process/_in_process.py", line 331, in main
      json_out['return_val'] = hook(**hook_input['kwargs'])
    File "/home/fabian/anaconda3/envs/test/lib/python3.7/site-packages/pip/_vendor/pep517/in_process/_in_process.py", line 117, in get_requires_for_build_wheel
      return hook(config_settings)
    File "/tmp/pip-build-env-ybm4ra2v/overlay/lib/python3.7/site-packages/setuptools/build_meta.py", line 155, in get_requires_for_build_wheel
      config_settings, requirements=['wheel'])
    File "/tmp/pip-build-env-ybm4ra2v/overlay/lib/python3.7/site-packages/setuptools/build_meta.py", line 135, in _get_build_requires
      self.run_setup()
    File "/tmp/pip-build-env-ybm4ra2v/overlay/lib/python3.7/site-packages/setuptools/build_meta.py", line 259, in run_setup
      self).run_setup(setup_script=setup_script)
    File "/tmp/pip-build-env-ybm4ra2v/overlay/lib/python3.7/site-packages/setuptools/build_meta.py", line 150, in run_setup
      exec(compile(code, __file__, 'exec'), locals())
    File "setup.py", line 27, in <module>
      long_description=read("README.md") + read("CHANGELOG.md"),
    File "setup.py", line 10, in read
      return io.open(file_path, encoding="utf-8").read()
  FileNotFoundError: [Errno 2] No such file or directory: 'CHANGELOG.md'
  ----------------------------------------
WARNING: Discarding https://files.pythonhosted.org/packages/e7/bc/8db10ddcadb9dd552baf40543c3b0a74029684c5dbd3880d1e8114b6d4ab/pyodc-1.1.0.tar.gz#sha256=34fd83c5bdeea578a5d0efb1c51b459e6b8ddaf94cd26be2e11c72fb39b68a99 (from https://pypi.org/simple/pyodc/). Command errored out with exit status 1: /home/fabian/anaconda3/envs/test/bin/python /home/fabian/anaconda3/envs/test/lib/python3.7/site-packages/pip/_vendor/pep517/in_process/_in_process.py get_requires_for_build_wheel /tmp/tmpiv0acpiq Check the logs for full command output.
ERROR: Could not find a version that satisfies the requirement pyodc==1.1.0 (from versions: 1.0.0, 1.0.1, 1.0.2, 1.0.3, 1.1.0)
ERROR: No matching distribution found for pyodc==1.1.0
```
I am actually not directly depending on `pyodc`, but I am using [climetlab](https://github.com/ecmwf/climetlab), whose newest version `0.8.18` depends on `pyodc==1.1.0` (i.e. it does not pin a specific version).

**Comment by b8raoult** [comment]
We will fix that ASAP. In the meantime, just 

`pip install pyodc==1.0.0`

That should do the trick.

**Comment by simondsmart** [comment]
This is fixed with the latest release, 1.1.1.
